xml/System/TimeOnly.xml

```diff
@@ -630,7 +630,15 @@ If `start` and `end` are equal, this method returns `false`, meaning there is ze
       <Docs>
         <summary>Represents the largest possible value of <see cref="T:System.TimeOnly" />.</summary>
         <value>A <see cref="T:System.TimeOnly" /> instance representing its largest possible value.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For <xref:System.IFloatingPoint`1> this method matches the IEEE 754:2019 `maximum` function. This requires `NaN` inputs to be propagated back to the caller and for `-0.0` to be treated as less than `+0.0`.
```

Comment:
Why we have such comment here? 

CC @tannergooding

Reply:
Looks like, for some reason, this has an `/// <inheritdoc` attached

Looks like an accidental merge conflict or find/replace error that should exist.

Reply:
It should be removed

Reply:
Thanks for clarifying, @tannergooding . Can you please address these in triple slash so this doesn't get ported again?